src/nginxconfig/i18n/pt-br/templates/domain_sections/rails.js

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2020 DigitalOcean
```

Comment:
```suggestion
Copyright 2021 DigitalOcean
```
Ditto -- 2021, please